site/content/docs/developing/addons/workload.en.md

```diff
@@ -0,0 +1,240 @@
+2. When running `copilot [svc/job] deploy`, the custom addons template will be deployed along with your workload stack.
+
+
+???- note "Sample workspace layout with workload addons"
+    ```term
+    .
+    └── copilot
+        └── webhook
+            ├── addons # Store addons associated with the service "webhook".
+            │   └── mytable-ddb.yaml
+            └── manifest.yaml 
+    ```
+
+## What does an addon template look like?
+A workload addon template can be [any valid CloudFormation template](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/template-anatomy.html) that satisfies the following:
+
+* Include at least one [`Resource`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/resources-section-structure.html).
+* The [`Parameters`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html) section must include `App`, `Env`, `Name`.
+
+you can customize your resource properties with [Conditions](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/conditions-section-structure.html) or [Mappings](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/mappings-section-structure.html).
+
+!!! info ""
+    We recommend following [Amazon IAM best practices](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html) while defining AWS Managed Policies for the additional resources, including:
+
+    * [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege) to the policies defined in your `addons/` directory.  
+    * [Use policy conditions for extra security](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#use-policy-conditions) to restrict your policies to access only the resources defined in your `addons/` directory.   
+
+
+### Writing the `Parameters` section
+
+There are a few parameters that Copilot requires you to define in your templates. 
+
+!!! info ""
+    ```yaml
+    # Required parameters for a workload addon by AWS Copilot.
+    Parameters:
+        App:
+            Type: String
+        Env:
+            Type: String
+        Name:
+            Type: String
+    ```
+
+
+#### Customizing the `Parameters` section
+
+If you'd like to define parameters in addition to the ones required by Copilot, you can do so with a
+`addons.parameters.yml` file.
+
+```term
+.
+└── addons/
+    ├── template.yml
+    └── addons.parameters.yml # Add this file under your addons/ directory.
+```
+
+1. In your template file, add the additional parameters under the `Parameters` section.
+2. In your `addons.parameters.yml`, define the values of those additional parameters. They can refer to values from your workload stack. 
+
+???- note "Examples: Customize addon parameters"
+    ```yaml
+    # In "webhook/addons/my-addon.yml"
+    Parameters:
+      # Required parameters by AWS Copilot.
```

Comment:
or "Parameters required by AWS Copilot."??

Reply:
I ended up just removing the comment for both workload and env addons because they seem redundant!